TRX/Tron-overview.md

```diff
@@ -72,51 +72,51 @@ DISK capacity depends on the actual transaction volume after deployment, but it
 ### 4.2.1 gRPC interface
 The Full Node and Solidity Nodes each run a gRPC service that you can connect to.
 
 - [gRPC Tutorial](https://grpc.io/docs/)
-- [gRPC API](https://github.com/TRONprotocol/Documentation/blob/master/English_Documentation/TRON_Protocol/TRON_Wallet_RPC-API.md)
+- [gRPC API](https://github.com/tronprotocol/Documentation/blob/master/English_Documentation/TRON_Protocol/TRON_Wallet_RPC-API.md)
 
-Please refer to the following two classes for a GRPC example in Java.
+Please refer to the following two classes for a gRPC example in Java.
 ```
-https://github.com/TRONprotocol/wallet-cli/blob/master/src/main/java/org/TRON/walletserver/WalletClient.java
+https://github.com/tronprotocol/wallet-cli/blob/master/src/main/java/org/tron/walletserver/WalletApi.java
 
-https://github.com/TRONprotocol/wallet-cli/blob/master/src/main/java/org/TRON/walletserver/GrpcClient.java
+https://github.com/tronprotocol/wallet-cli/blob/master/src/main/java/org/tron/walletserver/GrpcClient.java
 ```
 
 ### 4.2.2 HTTP Interface
 The FullNode and SolidityNode both have an HTTP Service running on them. All parameters are encoded as `HEX` and returned as Hex or `base58check`. If you're trying to pass an address in, please decode from `base58check` and conver it to `HEX`.
 
-- [HTTP Service API](https://github.com/TRONprotocol/Documentation/blob/master/TRX/TRON-http.md)
+- [HTTP Service API](https://github.com/tronprotocol/Documentation/blob/master/TRX/Tron-http.md)
 
-- [Address DEBUG Tool](https://github.com/tronprotocol/tron-demo/blob/master/TronConvertTool.zip)
+- [Address DEBUG Tool](https://github.com/tronprotocol/tron-demo/raw/master/TronConvertTool.zip)
 
 # 5. Transaction Fees
 Having too many transactions will clog our network like Ethereum and may incur delays on transaction confirmation. To keep the network operating smoothly, TRON network grants every account a free pool of `Bandwidth` for free transactions every 24 hours. To engage in transactions more frequently requires freezing TRX for additional bandwidth, or paying the fee in TRX.
 
-See also: https://github.com/TRONprotocol/Documentation/blob/master/English_Documentation/TRON_Protocol/Mechanism_Introduction.md
+See also: https://github.com/tronprotocol/Documentation/blob/master/English_Documentation/TRON_Protocol/Mechanism_Introduction.md
 
 ## 5.1 Definition of Bandwidth
 Transactions are transmitted and stored in the network in byte arrays. Bandwidth consumed in a transaction equals the size of its byte array.
 If the length of a byte array is 200 then the transaction consumes 200 bandwidth.
 
 ## 5.2 Freeze/unfreeze mechanism 
-TRX can be frozen for a minimum of 3 days to gain both `TRON Power` for voting and network `Bandwidth`. `TRON Power` is gained at a 1:1 ratio with the amount of frozen TRX.
+TRX can be frozen for a minimum of 3 days to gain both `TRON Power(TP)` for voting and `Bandwidth` for network. `TRON Power` is gained at a 1:1 ratio with the amount of frozen TRX.
```

Comment:
"`Bandwidth` for network" doesn't really make sense in my opinion. A better formed sentence might be "TRX can be frozen for a minimum of 3 days to gain both `TRON Power (TP)` for voting and `Bandwidth` for covering network fees.